EasyPost/Calculation/Rates.cs

```diff
@@ -76,29 +126,52 @@ public static Smartrate GetLowestShipmentSmartrate(IEnumerable<Smartrate> smartr
         {
             Smartrate? lowestSmartrate = null;
 
-            foreach (Smartrate? smartrate in from smartrate in smartrates let smartrateAccuracy = smartrate.TimeInTransit.GetBySmartrateAccuracy(deliveryAccuracy) where smartrateAccuracy != null where !(smartrateAccuracy > deliveryDays) select smartrate)
+            foreach (Smartrate? smartrate in smartrates)
             {
-                // if lowest smartrate is null, set it to this smartrate
+                if (smartrate.TimeInTransit == null)
```

Comment:
SmartRate will always have a Time in Transit property, that's what makes it a smartrate.

Reply:
But, what if it doesn't?